### DISCUSSION TITLE
What did you use for the official documentation?

### DISCUSSION THREAD

**Discussion by douglaslassance** [FlorianWoelki/obsidian-icon-folder#275] [Q&A] [answered]
The [documentation](https://florianwoelki.github.io/obsidian-iconize/) looks quite neat.

**Comment by FlorianWoelki** [comment, marked as answer]
Thank you :D It's vitepress.

**Reply by douglaslassance** [reply]
It's Node.js based. How do you make it static for GitHub Pages?

**Reply by douglaslassance** [reply]
I see, it can [deploy to static](https://vitejs.dev/guide/static-deploy.html).
